### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'replace')

### ISSUE THREAD

**Issue by armanozak** [elchininet/postcss-rtlcss#110]
First of all, thank you very much for this plugin. It has saved us a lot of time.

The `cleanRuleRawsBefore` function replaces multiple new lines from the `before` property of a node's `raws`, but results in a type error same as the title of this issue when the property is missing.

The error is caused by this line: https://github.com/elchininet/postcss-rtlcss/blob/master/src/utilities/rules.ts#L165

The following change fixes the error:

```
node.raws.before = '\n\n' + (node.raws.before ? node.raws.before.replace(/\n/g, '') : '');
```

Please let me know if you would like me to create a PR.

**Comment by elchininet** [comment]
Hi @armanozak, pull requests are always welcome. But it would be good to know when this happens to write the appropriate tests to cover it. Do you know exactly in which cases is this property missing?

**Comment by armanozak** [comment]
Hi @elchininet,

Thanks for the quick response. The error appeared when we upgraded Tailwind from v1 to v3 and only on directional classes with [responsive modifiers](https://tailwindcss.com/docs/responsive-design).

I have opened a PR. I saw the tests, but could not figure out how this change fits in the existing tests.

**Comment by elchininet** [comment]
Hey @armanozak, don't worry, just provide me with a minimum reproducible example of the issue, I'll check when and why it fails and I'll push the tests to your pull request.

Thanks!

**Comment by armanozak** [comment]
Hey @elchininet,

Here is a minimal reproduction of the issue: https://github.com/armanozak/repro-postcss-rtlcss-110

Thanks again!

**Comment by elchininet** [comment]
Perfect @armanozak, let it on my hands ;)
Thanks

**Comment by elchininet** [comment]
Hi @armanozak, it was impossible to create a test because it doesn‘t fail if the CSS is loaded directly from a file, it only fails if the CSS is built using tailwind, in these cases the rules built have an empty `raws` object. So, I‘ll merge the merge request (the tests are frozen for some reason) and try to make some tests of the method later to try to cover that case.

**Comment by armanozak** [comment]
Thank you @elchininet. Have a good evening.

**Comment by elchininet** [comment]
Hi @armanozak, your fix is contained in [version 3.5.1 of the package](https://www.npmjs.com/package/postcss-rtlcss/v/3.5.1). Regards and thanks for your contribution.
